zulipterminal/ui.py

```diff
@@ -139,7 +139,8 @@ def main_window(self) -> Any:
             urwid.Divider(div_char=div_char),
         ])
 
-        w = urwid.Frame(self.body, title_bar, focus_part='body')
+        w = urwid.Frame(self.body, title_bar, footer=urwid.Text("DEMO TEXT"),
```

Comment:
Do we need to use `DEMO TEXT`? I think it's better to leave it as an empty string